.github/workflows/check_pr_description.yml

```diff
@@ -0,0 +1,66 @@
+name: Checking PR description
```

Comment:
```suggestion
name: Check PR description
```

Prefer phrasing workflow descriptions as commands just like file names and function names